app/src/components/market/sections/market_pooling/user_pool_data.tsx

```diff
@@ -78,8 +78,12 @@ export const UserPoolData: React.FC<Props> = (props: Props) => {
     displayUserEarnings = compoundService.calculateCTokenToBaseExchange(baseCollateral, userEarnings)
     displayTotalEarnings = compoundService.calculateCTokenToBaseExchange(baseCollateral, totalEarnings)
   }
-  const userLiquidityFormatted = formatNumber(formatBigNumber(displayUserLiquidity, baseCollateral.decimals))
-  const poolTokensFormatted = formatNumber(formatBigNumber(displayPoolTokens, baseCollateral.decimals))
+  const userLiquidityFormatted = formatNumber(
+    formatBigNumber(displayUserLiquidity, baseCollateral.decimals, baseCollateral.decimals),
+  )
+  const poolTokensFormatted = formatNumber(
+    formatBigNumber(displayPoolTokens, baseCollateral.decimals, baseCollateral.decimals),
+  )
```

Comment:
Works good! I would just move these directly to value field in component property. Just to be consistent with the rest of code.